modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -120,8 +102,8 @@ object WebServerLauncher extends ServerApp with LogInitialization {
       _  <- configLog
       ac <- authConf
       wc <- serverConf
-      sc <- executorConf
-      _  <- seqexecExecutor.run(sc)
+      c <- config
+      _  <- SeqexecEngine.seqexecConfigure.run(c)
```

Comment:
Why not follow the existing pattern of configuration? Previously there was not need to do in-place mutations and wrap them into a `Task`. When running this you are just unwrapping the `Task` you just introduced in `seqexecConfigure`, but under the hood you already did an unsafe mutation outside of a `Task`.

I'm not saying this is outright wrong because maybe there is no way to around it, but I'm trying to understand what was the problem with the previous configuration pattern.